src/sec_edgar_bulker/core/models.py

```
@@ -0,0 +1,499 @@
+    """Settings for filing header parsing.
+    
+    Attributes:
+        use_config: Whether to use custom config.
+        patterns: Filing header patterns.
+    """
+    use_config: bool = False
+    patterns: FilingHeaderPatterns
+
+class DocumentStructure(BaseModel):
+    """Structure for document parsing."""
+    pass
```

Ruff (0.8.2)

Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)